javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,15 @@
+//getEventWeekDay returns the name of the day falling on given number of days
+function getEventWeekday(eventDay) {
+    const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    const today = new Date();
+    let noOfDays = today.getDay() + (eventDay % 7);
```

Comment:
Calculate the index of event day
```js
const dayIndex = (today.getDay() + daysLeft) % 7;
```
Then you can get the required day from the array by this index.